doc/source/sdk/proxies/index.rst

```diff
@@ -21,6 +21,7 @@ Service Proxies
    Object Block Storage (OBS) <obs>
    Relational Database Service RDS V1 (RDSv1) <rds_v1>
    Relational Database Service RDS V3 (RDS) <rds_v3>
+   Simple Message Notification Service (SMN) <smn>
```

Comment:
is it ok that version is omitted?

Reply:
yes, it is ok. Only in RDS it was necessary.